app/assets/stylesheets/utilities/_display.scss

```diff
@@ -36,3 +36,8 @@
 .faded {
   opacity: .5;
 }
+
+.fade-out {
+  opacity: 0;
+  transition: opacity 0.5s ease-out;
```

Comment:
Small nitpick; Everywhere `ms` is used instead of seconds.

Reply:
Will fix!